libraries/Microsoft.Bot.Connector/Teams/TeamsOperations.cs

```diff
@@ -692,16 +555,54 @@ public async Task<HttpOperationResponse<BatchOperationState>> GetOperationStateA
         /// <returns>
         /// A response object containing the state and responses of the operation.
         /// </returns>
-        public async Task<HttpOperationResponse<BatchFailedEntriesResponse>> GetPagedFailedEntriesAsync(string operationId, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        public async Task<HttpOperationResponse<BatchFailedEntriesResponse>> GetPagedFailedEntriesAsync(string operationId, Dictionary<string, List<string>> customHeaders = null, string continuationToken = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            if (string.IsNullOrEmpty(operationId))
+            {
+                throw new ValidationException(ValidationRules.CannotBeNull, nameof(operationId));
+            }
+
+            var invocationId = TraceActivity("GetFailedEntriesPaginated", new { OperationId = operationId }, cancellationToken);
```

Comment:
Why this doesn't match the method's name?

Reply:
I just kept the original tracing of this method
![image](https://github.com/southworks/botbuilder-dotnet/assets/122501764/c5b11193-eb67-423f-9d49-27e0b442d45e)